src/components/dashboard/Dashboard.js

```diff
@@ -156,6 +160,16 @@ const Dashboard = props => {
     [balance, headerLarge],
   )
 
+  const onPressCryptoLiteracyNovemberBanner = useOnPress(() => {
+    fireEvent(INVITE_BANNER)
+    navigation.navigate('Rewards')
+  }, [screenProps])
+
+  const listHeaderComponent = useCallback(
+    () => isCryptoLiteracyNovember && <CryptoLiteracyNovemberBanner onPress={onPressCryptoLiteracyNovemberBanner} />,
+    [isCryptoLiteracyNovember],
+  )
```

Comment:
1. no need to wrap, it's an overkill here
2. listHeaderComponent could be an jsx element, so you don't need to pass a function

```suggestion
  const listHeaderComponent = isCryptoLiteracyNovember ? <CryptoLiteracyNovemberBanner onPress={onPressCryptoLiteracyNovemberBanner} /> : null
```